site/content/docs/getting-started/custom-workspace.md

```diff
@@ -1,11 +1,13 @@
+# Set up custom workspace
```

Comment:
nit: Slight preference for "Set up a custom workspace" or "Set a custom workspace".

Reply:
I agree, I was worried changing it would adversely affect the table of contents. However, I've fixed both with a new commit, and one other awkward TOC entry. Others could do with the same consistency, but I'll address those with a new PR once I'm sure no internal links are broken.